selfdrive/navd/tests/test_map_renderer.py

```diff
@@ -110,6 +105,11 @@ def _setup_test(self):
     assert self.vipc.connect(False)
     self.vipc.recv()
 
+    # assert we aren't using the GPU
```

Comment:
this check is a good idea, but I'd put it in test_onroad instead

Reply:
would also split adding that test into a separate PR before this one

Reply:
done